product_template_partner_ref/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+{
+    "name": "Product Template Partner Ref",
```

Comment:
Confusing description, no references to Website Sale or e-commerce, neither search keyword. And name suggestion: `website_sale_product_search_extended_ref_barcode` (ouch, very long)